metals/src/main/scala/scala/meta/internal/metals/BloopServers.scala

```diff
@@ -195,6 +195,10 @@ final class BloopServers(
         )
         Future.successful(languageClient.metalsExecuteClientCommand(command))
 
+      case (item, _)
+          if item == Messages.BloopGlobalJsonFilePremodified.useGlobalFile =>
```

Comment:
```scala
case (Messages.BloopGlobalJsonFilePremodified.useGlobalFile, _) =>
```

Reply:
It would need to be a val unfortunately, so this doesn't work.